src/sentry/sentry_metrics/indexer/models.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any
+
+from django.db import connections, models, router
+
+from sentry.db.models import Model
+from sentry.db.models.fields.bounded import BoundedBigIntegerField
+
+
+class MetricsKeyIndexer(Model):  # type: ignore
+    __include_in_export__ = False
+
+    organization_id = BoundedBigIntegerField()
+    string = models.CharField(max_length=200)
```

Comment:
I am not sure I get how does the int autoincrementing field get created.

Reply:
it's because I'm inheriting from the `Model` class that sets the id to the `BigBoundedAutoField` (which uses the pseudo type bigserial and thats what tells postgres to create the sequence) https://github.com/getsentry/sentry/blob/197cbb0375c9259b422c85704578763a40384a92/src/sentry/db/models/base.py#L85-L86

Reply:
Is it valuable to have a `date_added` column here? Not sure how space conscious we need to be, but it's often helpful

Reply:
That's a good idea. The metrics team is going to build the part of the data model that manages TTL and expiration, so they should have that and the last read time as well. But they are still working on that, I don't think that part of the data model exists yet. Still having a date added can be useful.